### ISSUE TITLE
Support parsing ALTER { SEQUENCE | TABLE | TYPE | VIEW } .. OWNER TO ..

### ISSUE THREAD

**Issue by sreuland** [jOOQ/jOOQ#10105]
### Expected behavior and actual behavior:
running jooq codegen in maven with DDLDatabase generator against a sql script, the script has statements:

```sql
ALTER mytable OWNER to myrole
```

the jooq codegen throws:

```
[ERROR] Failed to execute goal org.jooq:jooq-codegen-maven:3.13.1:generate (jooq) on project tester: Error running jOOQ code generation tool: Error while exporting schema: ADD, ALTER, COMMENT, DROP, MODIFY, or RENAME expected: [61:32] ...TE RESTRICT
[ERROR] );
[ERROR]
[ERROR] ALTER TABLE mytable [*]OWNER TO 
```

It appears like this variant of ALTER with OWNER is causing the DDLDatabase parser to error out.

### Versions:

- jOOQ: 3.13.1
- Java:1.13
- Database (include vendor): Postgres
- OS: linux
- JDBC Driver (include name if inofficial driver):

**Comment by lukaseder** [comment]
Thank you very much for your suggestion. Indeed, it would be useful to parse (and ignore) this statement in the `DDLDatabase`. This is also listed here: https://github.com/jOOQ/jOOQ/issues/9775

In the future, we might support this statement also in the API (https://github.com/jOOQ/jOOQ/issues/10108), but I think supporting it in the parser is more immediately useful

**Comment by lukaseder** [comment]
We already do this for `ALTER SCHEMA .. OWNER TO`, but not for tables

**Comment by lukaseder** [comment]
Fixed for jOOQ 3.14.0. Backport scheduled for 3.13.2 (#10109).

Thanks again for your report. Notice, you can comment out unsupported statements for the jOOQ parser only, keeping them active for your PostgreSQL migrations, by using the comment syntax documented here:
https://www.jooq.org/doc/3.13/manual/sql-building/dsl-context/custom-settings/settings-parser/

Example:

```sql
CREATE TABLE t (i INT);

/* [jooq ignore start] */
ALTER TABLE t OWNER TO u;
/* [jooq ignore stop] */
```
